src/lib/utils.js

```diff
@@ -17,13 +17,15 @@ var DEFAULT_CONFIG_VALUES = {
   globals: {},
   moduleLoader: require.resolve('../HasteModuleLoader/HasteModuleLoader'),
   modulePathIgnorePatterns: [],
+  testCollector: require.resolve('../IstanbulCollector'),
```

Comment:
Let's call this `coverageCollector` to be more specific since it's only really collecting coverage information.